liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java

```diff
@@ -23,13 +23,15 @@
 import org.eclipse.lsp4j.ServerCapabilities;
 import org.eclipse.lsp4j.TextDocumentSyncKind;
 import org.eclipse.lsp4j.services.LanguageClient;
+import org.eclipse.lsp4j.services.LanguageClientAware;
 import org.eclipse.lsp4j.services.LanguageServer;
 import org.eclipse.lsp4j.services.TextDocumentService;
 import org.eclipse.lsp4j.services.WorkspaceService;
 
 
-public class LibertyLanguageServer implements LanguageServer {
+public class LibertyLanguageServer extends AbstractLanguageServer implements LanguageServer, LanguageClientAware {
```

Comment:
Will these changes affect how the various IDEs start/interact with the Liberty Language Server?

Reply:
I don't think they will change. They should still use the original initialization methods we had in place before, and these additions are in place for our test suite to startup the language server.

Reply:
The functionality looks different in the original initialization methods. That is why I asked this question.

Reply:
You may be looking at the initialization for the Test language server?